tools/codegen/api/python.py

```diff
@@ -841,7 +843,8 @@ def arg_parser_unpack_method(t: Type, has_default: bool) -> str:
             return 'intlist'
         elif str(t) == 'float[]':
             return 'doublelist'
-
+        elif str(t) == 'Scalar[]':
```

Comment:
Can user specify definite size for Scalar list? i.e. is it possible to have 'Scalar[3]' in function schema?
If so, please use `elif str(t.elem) == 'Scalar'` like how it handles `Dimname` and `int` above.

So why don't we handle `float[]` in the same way? The original intention is to inherit the same 'constraint' as the old codegen did, i.e. to make it fail loudly on 'float[3]' instead of silently accepting it. Thinking about it again, maybe it's over kill...

Reply:
No, user can not. For all APIs there is an overload for a single scalar and a list of scalars with no definite size.